cmp/grid/impl/GridPersistenceModel.js

```diff
@@ -189,14 +161,26 @@ export class GridStateModel {
         return ret;
     }
 
-    @debounced(500)
-    saveStateChange() {
-        const {state} = this;
+    @action
+    patchState(updates) {
+        this.state = {...this.state, ...updates};
```

Comment:
Is there anyway for old, unused state to be cleared out? For example, if we turn off `persistColumns`, it looks like columns will forever still be kept around as part of the state? (not saying we didn't have this situation already)